tools/validate_dme.py

```diff
@@ -84,7 +84,7 @@ def compare_lines(a, b):
         if a_segment != b_segment:
             return (a_segment > b_segment) - (a_segment < b_segment)
 
-    raise f"Two lines were exactly the same ({a} vs. {b})"
+    raise ValueError(f"Two lines were exactly the same ({a} vs. {b})")
```

Comment:
```suggestion
    print(f"Two lines were exactly the same ({a} vs. {b})")
    sys.exit(1)
```

We shouldn't be exposing exceptions to contributors, BTW ideally this would also include the line number itself but that's too complicated for my web edit